### PR TITLE
Change NP environment

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/488551667048375/1205234536443669/f

### Description
Change the NP environment

### Steps to test this PR
NetP smoke test

### CODE REVIEW

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3436** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3436" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3436?utm_source=stack-comment).

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgVpnControllerService.kt [hunk 1]
```diff
@@ -39,7 +39,7 @@ interface WgVpnControllerService {
     ): List<EligibleServerInfo>
 }
 
-const val NETP_ENVIRONMENT_URL = "https://staging.netp.duckduckgo.com"
+const val NETP_ENVIRONMENT_URL = "https://controller.netp.duckduckgo.com"
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgVpnControllerService.kt:42]: You need to add this in `WgVpnControllerRequestInterceptor`

Comment (karlenDimla) [reply network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgVpnControllerService.kt:42]: Having that said, we might need to integrate the `NetPInternalEnvDataStore` here as well especially if we are using a different endpoint. Completely missed this.

Comment (karlenDimla) [reply network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgVpnControllerService.kt:42]: If we don't do this, the auth token will not be added.

**Review by karlenDimla — APPROVED** [review]

**Pull request merged**